src/i18n/i18n.go

```diff
@@ -2,10 +2,11 @@ package i18n
 
 import "fmt"
 
-func T(textConst string, args ...interface{}) string {
+func T(textConst string, args ...any) string {
 	translation, exists := en[textConst]
 	if !exists {
-		return "[missing translation] " + textConst
+		format := textConst
```

Comment:
Is this necessary?

Reply:
Kinda, otherwise it breaks go vet's printf check. But I wanted to format `textConstant` variable because when prototyping it's handy to call this fucntion with draft of desired text and it's args and having `[missing translation]` prefix. Therefore I see which text need's to be yet included in i18n package. But I'm open to different solution.